helpers/pluck.js

```diff
@@ -1,10 +1,21 @@
 'use strict';
-
-var _ = require('lodash');
+const common = require('./lib/common.js');
 
 const factory = () => {
     return function(collection, path) {
-        return _.pluck(collection, path);
+        const arr = [];
+        if (Array.isArray(collection)) {
+            for (let i = 0; i < collection.length; i++) {
+                arr.push(collection[i][path]);
+            }
+        } else if (common.isObject(collection)) {
+            const keys = Object.keys(collection);
+            for (let i = 0; i < keys.length; i++) {
+                arr.push(collection[keys[i]][path]);
```

Comment:
Don't we need some kind of existence check here? Otherwise, we'll push `undefined` into the array.

Reply:
Based on documentation.  I thought it would be ok to not do that.  But I have changed it.